### DISCUSSION TITLE
During test, Audiogridder out performs Waves' Soundgrid server!!

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#742] [General]
I just needed to share this with everyone. I just tested a project using Audiogridder and Waves' Soundgrid server on the same machine for both servers.

In short, AG outperformed Waves when loading a lot of plugins in a chain (StudioRack chain). It seems that Waves SG server was designed for low latency and in that regard, it starts hitting a peak when some high buffer plugins are used in a chain.

 The project I was using for this test had around 8 tracks but on each track, I have around 4 plugins in a chain (around 32 loaded plugins). NOTE: I did have a few tape/reverb plugins on some tracks.
I tested the same project using AG (plus I added a few other non-waves plugins to the chain in AG plugin) and it handle it without issues using the default buffer configuration on AG!!! I tried adjusting SG network buffer but it didn't help.

**Comment by CaBuffalo** [comment]
I'm so happy to hear that. Thanks for letting us know about this.

Thanks, JaacMusic.

On Wed, Sep 29, 2021 at 10:41 AM jaacMusic ***@***.***> wrote:

> I just needed to share this with everyone. I just tested a project using
> Audiogridder and Waves' Soundgrid server on the same machine for both
> servers.
>
> In short, AG outperformed Waves when loading a lot of plugins in a chain
> (StudioRack chain). It seems that Waves SG server was designed for low
> latency and in that regard, it starts hitting a peak when some high buffer
> plugins are used in a chain.
>
> The project I was using for this test had around 8 tracks but on each
> track, I have around 4 plugins in a chain (around 32 loaded plugins). NOTE:
> I did have a few tape/reverb plugins on some tracks.
> I tested the same project using AG (plus I added a few other non-waves
> plugins to the chain in AG plugin) and it handle it without issues using
> the default buffer configuration on AG!!! I tried adjusting SG network
> buffer but it didn't help.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/742>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6F7ZR7YNA6XNJC2NFTUENFTHANCNFSM5FANW7YQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.
